dbx/issues-and-help.rst

```diff
@@ -0,0 +1,53 @@
+=============
+Issues & Help
+=============
+
+We are lucky to have a vibrant developer community that includes users that
+have various levels of experience using the {+driver-short+}. The quickest way
+to get support for general questions is through the `MongoDB Community Forums <https://www.mongodb.com/community/forums/>`__.
+
+To learn more about our support channels, see :manual:`</support/>` in the
+MongoDB Server documentation.
+
+Bugs / Feature Requests
+-----------------------
+
+To report a bug or request a new feature in the {+driver-short+}, open
+a case in the issue management tool, JIRA, by performing the following steps:
+
+1. Visit the `MongoDB JIRA issue tracker <https://jira.mongodb.org/>`__ and
+   `register or login <https://account.mongodb.com/account/register>`__.
+   Then, log in to JIRA.
+#. Navigate to the |JIRA-project-target|.
```

Comment:
Q: Will this work as a link?

Reply:
This is just a placeholder. In the page that includes this via ``sharedinclude``, this would get replaced by a ``replacement`` directive. As far as I understand, the replacement can be any rST, but I need to merge this first to properly test.